src/chart.zoom.js

```diff
@@ -158,21 +158,28 @@ function zoomScale(scale, zoom, center, zoomOptions) {
 	}
 }
 
-function doZoom(chartInstance, zoomX, zoomY, center, whichAxes) {
-	var ca = chartInstance.chartArea;
-	if (!center) {
-		center = {
+/**
+ * @param chart The chart instance
+ * @param {number} percentZoomX The zoom percentage in the x direction
+ * @param {number} percentZoomY The zoom percentage in the y direction
+ * @param {Object} focalPoint The x and y coordinates of zoom focal point. The point which doesn't change while zooming. E.g. the location of the mouse cursor when "drag: false"
```

Comment:
You could document the type of `focalPoint` as below:

```
@param {{x: number, y: number}} focalPoint
```

Reply:
Good idea. Done